docs/source/contents/4_draft_mapping.rst

```diff
@@ -2520,24 +2520,24 @@ which requires that the value is part of the class PeriodOfTime).
 
 .. code-block:: turtle
 
-    @prefix schema: <http://schema.org/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
 
-    <https://example.org/objects/1> schema:temporalCoverage "The Birth of Arrowmont, Gatlinburg, Tennessee, 1965-1979" .
+    <https://example.org/objects/1> dcterms:temporal "The Birth of Arrowmont, Gatlinburg, Tennessee, 1965-1979" .
 
-In addition to these textual values, UT does have :code:`temporal` :code:`subject`\ s that share numeric dates in EDTF format. These are
-primarily from the Volunteer Voices collection. `Here's an example record - volvoices:2945 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A2945/datastream/MODS/view>`_.
+In addition to these textual values, UT does have :code:`temporal` :code:`subject`\ s that share numeric dates in EDTF format. When a single
```

Comment:
```suggestion
In addition to these textual values, UT does have :code:`subject/temporal`\ s that share numeric dates in EDTF format. When a single
```
Refer to the XPATH rather than seperate node names. This is more clear.